core/CliMulti/RequestCommand.php

```diff
@@ -67,7 +67,13 @@ protected function execute(InputInterface $input, OutputInterface $output)
         }
 
         if ($input->getOption('superuser')) {
-            Access::getInstance()->setSuperUserAccess(true);
+            StaticContainer::addDefinitions(array(
+                'observers.global' => \DI\add(array(
+                    array('Environment.bootstrapped', function () {
+                        Access::getInstance()->setSuperUserAccess(true);
+                    })
+                )),
+            ));
```

Comment:
The container is created in Console, then RequestCommand creates a new one for RequestCommand stuff (mainly because the Cache is created in UrlHelper), then it includes index.php which creates the environment again. And the last environment creation is where we want to set superuser access.

When the web app frontend is encapsulated, it will look a lot better, eg:

```
$webApp = new WebApplication();
$webApp->getEnvironment()->getContainer()->get('Piwik\Access')->setSuperUserAccess(true);
$webApp->dispatch(...);
```